.github/workflows/checkbrname.yml

```diff
@@ -11,9 +11,9 @@ jobs:
   formalities:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v3
       - name: Set up Python 3.7
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v3
```

Comment:
Does this action even work? 
Isn't it supposed to add the PRs to the issues?

Reply:
Oh I realized just now, seems it was all inactive but dangling there. This is strange.